cmd/dagger/main.go

```diff
@@ -25,6 +29,7 @@ func init() {
 
 	rootCmd.PersistentFlags().StringVar(&workdir, "workdir", ".", "The host workdir loaded into dagger")
 	rootCmd.PersistentFlags().BoolVar(&debugLogs, "debug", false, "show buildkit debug logs")
+	rootCmd.PersistentFlags().StringVar(&cpuprofile, "cpuprofile", "", "collect CPU profile to path, and trace at path.trace")
```

Comment:
🆒

Reply:
🤯

Reply:
Just curious, how do you work with Go `*.trace` files @vito ?

Reply:
I was using this: https://github.com/felixge/traceutils